Akhil forgotpassword

Aillgates opened the pull request:
. Create a "Forgot Password" page with form validation.

. Build a reusable modal for success/error messages.

. password reset page

Aillgates committed:
added forgot password 2 pages and model

Aillgates committed:
changed forgot page

JaseelaAlimahamed commented on src/App.jsx:
```diff
@@ -1,12 +1,12 @@
-import UserRoutes from './routes/UserRoutes';
+import { Routes, Route } from "react-router-dom";
+import ProviderRoutes from "./routes/ProviderRoutes.jsx";
+import Hello from "./Hello.jsx";
```
./hello.jsx means?

Aillgates committed:
Update App.jsx

I removed the hello.jsx import that was initially created for testing the component.

JaseelaAlimahamed committed:
Merge branch 'develop' into akhilForgotpassword

JaseelaAlimahamed commented on src/routes/ProviderRoutes.jsx:
```diff
@@ -1,5 +1,8 @@
 import { Route, Routes, Navigate } from 'react-router-dom'
+import NewPassword from '../pages/NewPassword'
+import ForgotPassword from '../pages/ForgotPassword'
+import OTPPage from '../pages/ForgotOtp'
```
why forgot otp
?????

Aillgates replied:
I have developed a 'Forgot OTP Password' page, and now realize it should be rendered upon clicking the designated button.

JaseelaAlimahamed replied:
forgot otp fuction is not there either forgot password or resend otp

JaseelaAlimahamed commented on src/components/reUsableComponents/ButtonComponent.jsx:
```diff
@@ -1,17 +1,16 @@
 
-const ButtonComponent = ({ label, type = 'button', disabled = false, btnWidth = 'w-auto', btnHeight = 'h-auto' }) => {
+
+const ButtonComponent = ({ label, type, disabled, btnWidth , btnHeight , handleSubmit }) => {
```
btnWidth  btnHeigh why you change this reusable feeatures in button component

JaseelaAlimahamed committed:
Merge branch 'develop' into akhilForgotpassword
